desktop/apps/artist/client/views/articles.coffee

```diff
@@ -5,7 +5,7 @@ ArtworkRailView = require '../../../../components/artwork_rail/client/view.coffe
 template = -> require('../../templates/sections/articles.jade') arguments...
 sd = require('sharify').data
 query = require '../../queries/articles.coffee'
-metaphysics = require '../../../../lib/metaphysics.coffee'
+metaphysics = require '../../../../../lib/metaphysics.coffee'
```

Comment:
All of these changes to the relative paths had to go up one level everywhere, since it moved from a root-level `lib/` folder within desktop and mobile, to project-level `lib/` (up one from the desktop/mobile location).